test/integration/api/api_test.go

```diff
@@ -88,14 +88,13 @@ func TestAPI_Metrics(t *testing.T) {
 	resp, err := client.R().EnableTrace().Get(url)
 	assert.NoError(t, err)
 	assert.Equal(t, http.StatusOK, resp.StatusCode())
-	assert.Contains(t, resp.String(), "system_cpu_system")
+	//assert.Contains(t, resp.String(), "system_cpu_system")
```

Comment:
Can you add this back in?

Reply:
will do

Reply:
think you may have forgot to add this back in

Reply:
thanks, done.